config/software/knife-spork.rb

```diff
@@ -17,6 +17,9 @@
 name "knife-spork"
 default_version "master"
 
+license "TBD"
```

Comment:
This should be Apache-2.0 based on https://github.com/jonlives/knife-spork/blob/master/LICENSE.

It has some NOTICE additions which are common in this license.